DAO/MessageDAO.py

```diff
@@ -57,3 +59,48 @@ def like_message(self, mid):
 
     def dislike_message(self, mid):
         pass
+
+    def get_num_messages_daily(self, date):
+        cursor = self.get_cursor()
+        end_date = date + relativedelta(days=1)
+        query = "select count(*) as num from messages where created_on > %s and created_on < %s"
+        cursor.execute(query, (date, end_date))
+        count = [row for row in cursor]
```

Comment:
With RealDictCursor all we have to do is 
```python
return cursor.fetchall()
```

Reply:
Oh perfect that's exactly what I was looking for